misc/describe-options.txt

```diff
@@ -0,0 +1,109 @@
+# Short (less than one line) descriptions of tarsnap options.
+#
+# Format:
+#     --option \t+ description [\t+ (MODE|DUP-.*)]
+
+--aggressive-networking	use multiple TCP connections
+--archive-names		read a list of archive names from a file
+--cachedir		specify cache directory
+--check-links		warn unless all links to each file are archived
+--checkpoint-bytes	add a checkpoint after every ARG bytes of uploaded data
+--chroot		chroot to the current directory after -C options
+--configfile		add file to the list of configuration files to be read
+--creationtime		manually specify a creation time for the archive
+--csv-file		write statistics in CSV format to a file
+--disk-pause		often pause for ARG ms while creating an archive
+--dry-run		do not really create an archive; just simulate doing so
+--dump-config		print all config lines
+--exclude		do not process certain files or directories
+--force-resources	force the decryption of a passphrase-encrypted keyfile
+--fsck			perform some integrity checks and reconstruct cache	MODE
+--fast-read		stop after the first entry which matches the ARG	DUP-q
+--fsck-prune		run as --fsck, but prune any corrupt archives	MODE
+--humanize-numbers	use SI prefixes for --print-stats
+--include		process only certain files or directories
+--initialize-cachedir	create and initialize the cachedir		MODE
+--insane-filesystems	allow descent into synthetic filesystems such as procfs
+--iso-dates		print dates as yyyy-mm-dd hh:mm:ss
+--keep-going		keep going after finding an archive which does not exist
+--keep-newer-files	keep files which are newer than the extracted files
+--keyfile		specify key file
+--list-archives		print the names of archives stored		MODE
+--lowmem		reduce memory usage by not caching small files
+--maxbw			stop if more than ARG bytes of upstream bw is used
+--maxbw-rate		limit bandwidth to ARG bytes per second
+--maxbw-rate-down	limit download bandwidth to ARG bytes per second
+--maxbw-rate-up		limit upload bandwidth to ARG bytes per second
+--newer			only add files and dirs newer than the given date
+--newer-mtime		like --newer, but it uses mtime instead of ctime
+--newer-mtime-than	like --newer-than, but it uses mtime instead of ctime
+--newer-than		only include files and dirs newer than the given file
+--no-aggressive-networking	ignore any aggressive-networking option
+--no-config-exclude	ignore any exclude option specified
```

Comment:
Not sure why "ignore any ... option specified in a configuration file" is sometimes shortened to "... option" and sometimes to "... option specified"?